carina-webdriver/src/test/java/com/qaprosoft/carina/core/foundation/webdriver/core/capability/impl/desktop/DesktopCapabilitiesTest.java

```diff
@@ -0,0 +1,253 @@
+package com.qaprosoft.carina.core.foundation.webdriver.core.capability.impl.desktop;
+
+import com.qaprosoft.carina.core.foundation.commons.SpecialKeywords;
+import com.qaprosoft.carina.core.foundation.utils.Configuration;
+import com.qaprosoft.carina.core.foundation.utils.R;
+import com.qaprosoft.carina.core.foundation.webdriver.core.capability.impl.windows.WindowsCapabilities;
+import org.openqa.selenium.Platform;
+import org.openqa.selenium.Proxy;
+import org.openqa.selenium.firefox.FirefoxProfile;
+import org.openqa.selenium.ie.InternetExplorerDriver;
+import org.openqa.selenium.remote.BrowserType;
+import org.openqa.selenium.remote.CapabilityType;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.*;
+
+public class DesktopCapabilitiesTest {
+
+    // default FirefoxProfile preferences
+    private static final boolean MEDIA_EME_ENABLED = true;
+    private static final boolean MEDIA_GMP_MANAGER_UPDATE_ENABLED = true;
+
+    @Test(groups = {"DesktopCapabilitiesTestClass"})
```

Comment:
I don’t see any benefits from declared group

Reply:
We need to add these groups to specify the order in which the tests will run. If they are not specified in this order, then when building the project locally tests are failed.